File: 3536-maximum-product-of-two-digits/3536-maximum-product-of-two-digits.cpp
class Solution {
public:
    int maxProduct(int n) {
        vector<int> vec;
        while(n>0){
            vec.push_back(n % 10);
            n/=10;
        }
        sort(vec.begin(),vec.end());

        return vec.back() * vec[vec.size()-2];
    }
};

File: 3536-maximum-product-of-two-digits/3536-maximum-product-of-two-digits.py
class Solution:
    def maxProduct(self, n: int) -> int:
        l = []
        while n>0:
            l.append(n % 10)
            n = n // 10
        mx_pro = l[0]*l[1]
        for i in range(len(l)):
            for j in range(i+1,len(l)):
                mx_pro = max(mx_pro , l[i]*l[j])
        
        return mx_pro